src/Microsoft.Azure.SignalR.Common/ServiceConnections/MultiEndpointServiceConnectionContainer.cs

```diff
@@ -231,13 +231,13 @@ internal IEnumerable<ServiceEndpoint> GetRoutedEndpoints(ServiceMessage message)
                     return _router.GetEndpointsForConnection(closeConnectionMessage.ConnectionId, endpoints);
 
                 case ClientInvocationMessage clientInvocationMessage:
-                    return SingleOrNotSupported(_router.GetEndpointsForConnection(clientInvocationMessage.ConnectionId, endpoints), clientInvocationMessage);
+                    return _router.GetEndpointsForBroadcast(endpoints);
```

Comment:
Shall we call `GetEndpointsForConnection`? Considering the client invocation should only send to the target client connection?